### PR TITLE
March 08

### PR DESCRIPTION
**Pull request by g-may**

### CODE REVIEW

File: entity-types-v2.md [hunk 1]
```diff
@@ -0,0 +1,485 @@
...
+{:tip: .tip}
+{:pre: .pre}
+{:codeblock: .codeblock}
+{:screen: .screen}
+{:javascript: .ph data-hd-programlang='javascript'}
+{:java: .ph data-hd-programlang='java'}
+{:python: .ph data-hd-programlang='python'}
+{:swift: .ph data-hd-programlang='swift'}
+
+# Entity types and subtypes (v2)
```
Comment (rhorsfal) [inline entity-types-v2.md]: Why v1 and v2 - The usage is based on a version date, this could be confusing.

File: entity-types.md [hunk 2]
```diff
@@ -19,471 +19,47 @@ lastupdated: "2017-10-04"
...
-| Writer                              |
 
+## Entity type systems for version 2018-03-08
+{: #2018-03-08}
+
+The following entity type systems are used when you set the `version` parameter to `2018-03-08`.
+
+|Language|Entity type system|
+| --- | ---|
+| English | [v1] |
```
Comment (rhorsfal) [inline entity-types.md]: English is v1 for both dates?

File: entity-types.md [hunk 3]
```diff
@@ -19,471 +19,47 @@ lastupdated: "2017-10-04"
...
+| German | [v1] |
+| Italian | [v1] |
+| Korean | [v1] |
+| Portuguese | [v1] |
+| Russian | [v1] |
+| Spanish | [v1] |
+| Swedish | [v1] |
+
+
+## Entity type systems for version 2017-02-27
```
Comment (rhorsfal) [inline entity-types.md]: version xxx and earlier?

File: toc [hunk 4]
```diff
@@ -22,8 +22,11 @@ Natural Language Understanding
     categories.md
     detectable-languages.md
     entity-types.md
+        entity-types-v1.md
```
Comment (rhorsfal) [inline toc]: Does this work? I thought it needed to be in a ".topicgroup"

File: usage-limits.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+    "concepts": {}
+  }
+}
+```
+
+## Concurrent request limit
+
+Each {{site.data.keyword.nlushort}} service instance is limited to 30 concurrent requests. It's possible that this limit may decrease for service instances on the Lite and Standard plans when the system is experiencing exceptionally heavy traffic. For example, an application that makes 35 simultaneous requests from a single service instance will exceed the concurrent request limit by five requests, and five of those requests will return a `429: Too Many Requests` error.
+
+To increase your concurrent request limit, [open a support ticket](https://ibm.biz/ibmcloudsupport).
```
Comment (rhorsfal) [inline usage-limits.md]: Do we really want to give them an explicit way to do this? Perhaps a contact sales if you need more than the standard service plans offer? Support seems like it's "something is wrong" issue, where it's really a "you're not paying enough to tax the system this much" issue.

File: usage-limits.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+
+## Language support
+
+Different language restrictions apply depending on how you use the service. For details, see the [Language support](language-support.html) page.
+
```
Comment (rhorsfal) [inline usage-limits.md:53]: Are there any "usage limit" language support items or is this just to make sure they know all the "Limitations"?

Comment (g-may) [reply usage-limits.md:53]: It's just a general limitation note

File: usage-limits.md (hunk 5 above)
Comment (g-may) [reply usage-limits.md]: Ran this by OM, initially I had "upgrade to Premium" but they can work with you even on the Standard plan to increase the limit if you open a support ticket.

**Pull request merged**